applications/luci-app-throughput-measurements/root/usr/share/luci/menu.d/luci-app-throughput-measurements.json

```diff
@@ -0,0 +1,41 @@
+{
+	"admin/troubleshooting": {
+		"title": "Troubleshooting",
+		"order": 80,
+		"action": {
+			"type": "firstchild"
+		}
+	},
+
+	"admin/troubleshooting/tputmeas": {
+		"title": "Throughput Measurements",
+		"order": 1,
+		"action": {
+			"type": "alias",
+			"path": "admin/troubleshooting/tputmeas/client"
+		},
+		"depends" : {
+			"acl": [ "luci-app-throughput-measurements" ],
+			"uci": { "tputmeas": true },
+			"fs": { "/usr/libexec/luci.tputmeas": "executable" }
```

Comment:
I can not find a rpcd in the directory

Reply:
Forgot to update path with rpcd directory